docs/howto/error-reporting.txt

```diff
@@ -255,13 +255,6 @@ following methods:
     Returns ``True`` to activate the filtering operated in the other methods.
     By default the filter is active if :setting:`DEBUG` is ``False``.
 
-.. method:: SafeExceptionReporterFilter.get_request_repr(request)
```

Comment:
Personally, I think that option 2 doesn't actually solves anything. Any part of the request or report may contain sensitive data. For example `request.COOKIES` or a setting which doesn't match `HIDDEN_SETTINGS` pattern. Even the link itself can contain sensitive data, for example if an error occurs in `password_reset_confirm` view the URL contains token which can be used to take over user's account.